revolv/templates/administrator/dashboard.html

```diff
@@ -3,143 +3,148 @@
 
 {% block title %}Home{% endblock %}
 
-{% block body %}
+{% block head %}
+  <link rel="stylesheet" type="text/css" href="{% static "dashboard.css" %}" />
+{% endblock %}
 
-<div class='row'>
-    <a href="{% url "donor:dashboard" %}">
-        <div class='medium primary dashboard-button button'>Donor Dashboard</div>
-    </a>
-    {% if user.revolvuserprofile.is_administrator %}
-    <a href="{% url "administrator:dashboard" %}">
-        <div class='medium active dashboard-button button'>Administrator Dashboard</div>
-    </a>
-    {% endif %}
-    {% if user.revolvuserprofile.is_ambassador %}
-    <a href="{% url "ambassador:dashboard" %}">
-        <div class='medium primary dashboard-button button'>Ambassador Dashboard</div>
-    </a>
-    {% endif %}
-</div>
+{% block javascripts %}
+  <script type="text/javascript" src="{% static "js/base/dashboard.js" %}"></script>
+{% endblock %}
 
-<div class='row text-center'>
-    <a href="{%url "administrator:dashboard" %}">
-        <div class='small active administrator-button button'>Projects</div>
-    </a>
-    <div class='small primary administrator-button button'>Repayment</div>
-    <div class='small primary administrator-button button'>Ambassadors</div>
-    <a href="{%url "administrator:email" %}">
-        <div class='small primary administrator-button button'>Emails</div>
-    </a>
-</div>
+{% block body %}
 
-<div class='row'>
-    <div class='small-3 columns dash-box'>
-        <dl class="tabs vertical-dash" data-tab>
-            {% for project in all_projects %}
-            {% if project in drafted_projects and project == drafted_projects.first %}
-            <h4 class='subheader text-center'>Drafted Projects</h4>
-            {% elif project in proposed_projects and project == proposed_projects.first %}
-            <h4 class='subheader text-center'>Proposed Projects</h4>
-            {% elif project in active_projects and project == active_projects.first %}
-            <h4 class='subheader text-center'>Active Projects</h4>
-            {% elif project in completed_projects and project == completed_projects.first %}
-            <h4 class='subheader text-center'>Completed Projects</h4>
-            {% endif %}
-            {% if project.id == active_project %}
-            <dd class="active proj-button"><a href="#proj{{project.id}}">{{ project.title }}</a></dd>
-            {% else %}
-            <dd class="proj-button"><a href="#proj{{project.id}}">{{ project.title }}</a></dd>
+<div class="row">
+      <div class='row'>
+            {% if user.revolvuserprofile.is_administrator %}
```

Comment:
You could simplify this template logic: right now the donor and ambassador dashboard links are repeated